_data/hosting.yml

```diff
@@ -425,6 +425,17 @@ websites:
     exception: "Only available for the main client area. Hosting panels not supported."
     regions:
       - se
+      
+  - name: OMGServ
+    url: https://www.omgserv.com
+    img: omgserv.png
+    tfa:
+      - totp
+    doc: https://www.omgserv.com/fr/faq-site/comment_activer_la_double_authentification-103/
+    exception: "Documentation only available in French, but illustrated"
```

Comment:
With the region tag I think it's unnecessary to include this exception note.

Thanks for being so thorough though! 🙂

Reply:
You're right. I've deleted the exception line :)